test/test_robot_description_loader.py

```diff
@@ -79,3 +79,12 @@ def test_use_gazebo_chest_camera():
     rdl.gz_control_config_package = 'sciurus17_description'
     rdl.gz_control_config_file_path = 'config/dummy_controllers.yaml'
     assert 'reference="chest_camera_link"' in exec_load(rdl)
+
+
+def test_use_mock_components():
+    # use_gazeboが変更され、xacroにgz_ros2_controlがセットされることを期待
```

Comment:
細かいですがコメントがコピペされたままだったので確認と修正お願いします
```suggestion
    # use_mock_componentsが変更され、xacroにmock_componentsがセットされることを期待
```

Reply:
ありがとうございます
適用しました